homebrew/install_omero51

```diff
@@ -14,6 +14,9 @@ export HTTPPORT=${HTTPPORT:-8080}
 
 cd /usr/local
 
+# Install Genshi (OMERO and Bio-Formats requirement)
+bin/pip install -U genshi
```

Comment:
Absolutely but the 5.1 formula is testing 5.1.0-m3 for now. I will open a PR to remove this when we release 5.1.0-m4